scala/lambdas/ingest-flow-control/src/main/scala/uk/gov/nationalarchives/ingestflowcontrol/Lambda.scala

```diff
@@ -122,16 +161,30 @@ class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
     }
 
     for {
+      _ <- IO(logger.info(Map("executionName" -> s"""${potentialInput.map(_.executionName)}""")(s"Starting flow control lambda with input: $potentialInput")))
       flowControlConfig <- dependencies.ssmClient.getParameter[FlowControlConfig](config.configParamName)
       _ <- writeItemToQueueTable(flowControlConfig)
       runningExecutions <- dependencies.stepFunctionClient.listStepFunctions(config.stepFunctionArn, Running)
+      _ <- IO(
+        logger.info(
+          Map("executionName" -> s"""${potentialInput.map(_.executionName)}""", "running executions" -> s"""${runningExecutions.map(_.mkString(","))}""")(
```

Comment:
You can just do `runningExecutions.mkString(",")` here.

Reply:
yeah, spotted that when making other change, in fact the existing statement was totally wrong as `mkString` was being applied to each element 🤦🏼‍♂️